buf.yaml

```diff
@@ -1,5 +1,10 @@
 # For details on buf.yaml configuration, visit https://buf.build/docs/configuration/v2/buf-yaml
 version: v2
+modules:
+  - path: .
+    excludes:
+      - node_modules
+      - website/node_modules
```

Comment:
maybe we should also add the docs node_modules? infact can it not just ignore all things in git ignore?